sklearn/impute.py

```diff
@@ -272,6 +285,11 @@ def fit(self, X, y=None):
                                                self.missing_values,
                                                fill_value)
 
+        if self.add_indicator:
+            self.indicator_ = MissingIndicator(
```

Comment:
`self.indicator_` should be `None` if `self.add_indicator = False`. 

This currently fails:

```
SimpleImputer().fit(X).indicator_
```

with `AttributeError: 'SimpleImputer' object has no attribute 'indicator_'`

Also please update the attribute docstring with "[...] None if add_indicator is False."